lib/ansible/module_utils/network/nxos/config/l3_interfaces/l3_interfaces.py

```diff
@@ -92,6 +97,8 @@ def set_config(self, existing_l3_interfaces_facts):
         if config:
             for w in config:
                 w.update({'name': normalize_interface(w['name'])})
+                if w['name'] == 'mgmt0':
```

Comment:
@mikewiebe Can you update the check to interface_type being management instead of hardcoding mgmt0?

Reply:
@trishnaguha Just to be clear, you are referring to this right?

https://github.com/ansible/ansible/pull/62545/files#diff-59e3b330fca2b94d60b98f37448fbd3aR85

Reply:
Yes that's correct

Reply:
Done